Release 0.91 doc update (#2407)
RedisLabs/redislabs-docs: content/rdi/data-transformation/_index.md

---
Title: Data transformation
linkTitle: Data transformation
description:
weight: 30
alwaysopen: false
categories: ["redis-di"]
aliases: 
---

The key functionality that RDI performs is mapping the data coming from [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html) (representing a Source Database row data or row state change) into a Redis key with a value of [Hash](https://redis.io/docs/data-types/hashes/) or [JSON](https://redis.io/docs/stack/json/).
There are two types of Data Transformations in RDI:

- By default, each source row is converted into one [Hash](https://redis.io/docs/data-types/hashes/) or one JSON key in Redis.

>   This conversion is using the Debezium Schema based conversion: the incoming data includes the schema and RDI will use a set of handlers to automatically convert each source column to a the a Redis Hash field or JSON type based on the Debezium type in the schema. [Look here](../reference/data-types-conversion/data-types-conversion.md) for a full reference of these conversions.

Reviewer: Typo fix and a few other suggestions. Also maybe we can rework the "Look here" sentence a bit.
```suggestion
  This conversion uses the Debezium schema-based conversion: the incoming data includes the schema and RDI uses a set of handlers to automatically convert each source column to a Redis Hash field or JSON type based on the Debezium type in the schema. [Look here](../reference/data-types-conversion/data-types-conversion.md) for a full reference of these conversions.
```
